### ISSUE TITLE
Self-signed apps don't install

### ISSUE THREAD

**Issue by Sn0wCh1ld** [Sn0wCh1ld/App-Installer#7]
Though CreatureSurvive has had some success installing apps signed with a free developer account, I tested it again, and it still doesn't work. I will update README.md accordingly.

**Comment by CreatureSurvive** [comment]
hmm, strange. I originally thought it was only working due to AppSync Unified but after uninstalling and rebooting I'm still successful in installing any of my apps signed with my free dev account.

**Comment by Sn0wCh1ld** [comment]
And you're installing those apps through App Installer? That's quite strange. Having AppSync Unified enabled *should* make it work, but without it, I have no idea why it works for you but not the rest of us.

**Comment by CreatureSurvive** [comment]
Yes, I'm installing with App-Installer. It could be because I already have a profile for my dev account on my device, but I'd imagine that you do as well. I'll do some testing and try to break it on my end, maybe that will guide us to a solution.

**Comment by Sn0wCh1ld** [comment]
I created a new Apple ID which I used to sign the apps with a free developer account (as my main account is paid). Maybe you could try that too?

**Comment by CreatureSurvive** [comment]
looks like my success was due to Extender: Reloaded by Matchstic. it allows installing locally provisioned apps which Apple does not allow OTA installs of locally provisioned apps. This isn't so much an issue as a limitation by Apple. I attempted to implement a fix using AutoHook to avoid MobileSubstrate dependency, but I haven't had any luck yet.

**Comment by Sn0wCh1ld** [comment]
Sorry, I don't know what AutoHook is, and a quick Google search didn't return much. Could you explain what it is?

**Comment by CreatureSurvive** [comment]
Ahh, my bad. I should have linked it. It's a portion of JohnCoates hooking framework. 

Simple Objective-C Method Hooking
https://gist.github.com/c0d77f130d033b206367db480f7c18ae

**Comment by Sn0wCh1ld** [comment]
Thanks. Wouldn't that require a jailbreak though, as that function is not built in to App Installer? Or is there another way/I misunderstood this?

**Comment by CreatureSurvive** [comment]
Actually, your right. I didn't even think about how this wouldn't work within a sandboxed process.

**Comment by Sn0wCh1ld** [comment]
Is there any other way to do it?

**Comment by CreatureSurvive** [comment]
As far as I'm aware of no, nothing I can find in the public API would allow this. Looking at some of Extender: Reloaded code it's possible there may be a flag you can use in the info.plist of the app to be installed, but I'm not sure how that works exactly.

**Comment by Sn0wCh1ld** [comment]
I mean, we don't actually have to stick to only the public API, since we're not submitting to the App Store. I don't know much about the private API, though I'd love to learn. How did you access the Extender: Reloaded code though? Isn't it closed-source?
